iFixit/react-commerce
# Add `Intl.RelativeTimeFormat` polyfill

Polyfills `Intl.RelativeTimeFormat` to prevent noisy sentry errors in older browsers.

We'll use our self-hosted polyfill in production because the EU doesn't like us using 3rd party JS (see https://github.com/iFixit/ifixit/issues/36983).

The script needs to be `beforeInteractive` so that we polyfill this constructor before our Next.js code runs.
https://nextjs.org/docs/api-reference/next/script#beforeinteractive

I generated the polyfill url using polyfill.io's builder: https://polyfill.io/v3/url-builder/

## QA

Make sure the polyfill script runs successfully in the network tab.

If you can, use an old version of Safari (13.1 or older) and make sure we don't get an error about `Intl.RelativeTimeFormat`.
https://caniuse.com/mdn-javascript_builtins_intl_relativetimeformat_relativetimeformat

Closes #963
